packages/suite/src/components/suite/Settings/components/Row.tsx

```diff
@@ -1,25 +1,19 @@
-import { HTMLAttributes } from 'react';
+import { ReactNode } from 'react';
 import styled from 'styled-components';
 
-const Content = styled.div`
+const Container = styled.div`
     display: flex;
     flex: 1;
-    justify-content: space-between;
-    padding: 16px 0px;
-`;
+    padding-bottom: 16px;
```

Comment:
Here and other places:

```suggestion
    padding-bottom: ${spacingsPx.md};
```

Reply:
I am trying to minimize changes to suite and i am not sure it will stay but why not :)  https://github.com/trezor/trezor-suite/pull/9374/commits/83712217cd967dc4b95e6652aee4f077424aa2b2